src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/processor/IClusterRegionProcessor.java

```diff
@@ -84,6 +84,7 @@ private Type getType() {
 
     }
 
+    @SuppressWarnings("java:S107")
```

Comment:
Suppressing Sonar issue complaining that method has 8 params instead of recommended 7 or less.

Reply:
Hi @AdilOoze Instead, we could create a DTO object and pass it as an argument. Sorry, I have written this method before. I should have taken care of it.

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-data-load/pull/176/commits/59acb866eaad76cae7cc0fa6c0f0536d9ccf6c0c